episodes/17-operations.md

```diff
@@ -35,22 +20,143 @@ exercises: 5
+Now it is time to consider the context in which this lesson development can take place.
+
+## The Carpentries Lesson Development Ecosystem
+
+[The Carpentries Incubator][carpentries-incubator] is a space for The Carpentries community
+to collaborate on lessons.
+It provides a single location where our community can come to 
+find lessons that are being developed by the community,
+and to create new lessons using The Carpentries Workbench.
+Community members retain ownership of their lessons hosted in the Incubator,
+while benefiting from the increased findability it gives to the lesson.
+
+The Carpentries supports the lesson developer community working in the Incubator
+with training (like this one),
+communications designed to raise awareness of lesson projects within the community,
+and platforms for the community to discuss and collaborate on lesson development.
+
+The Carpentries also provides a space for community-developed lessons
+to reside after the initial phases of development have been completed.
+[The Carpentries Lab][carpentries-lab] hosts a repository for open peer-review of lessons,
+and a collection of lessons that have passed through this review process 
+and can be considered ready to teach by any Instructor with sufficient knowledge of the topic.
+In the future, this lesson review process may also be used to determine when a lesson
+in the Incubator is ready to be incorporated into one of The Carpentries _Lesson Programs_,
+the collection of lessons and curricula that can be taught in an official workshop for
+Software, Library, or Data Carpentry.
+
+
+## The Lesson Life Cycle
+
+To make it easier for other community members to 
+assess the current state of a lesson in the Incubator,
+The Carpentries encourages lesson developers to indicate the status of their lesson 
+by labeling its progress through a _lesson life cycle_:
+
+![The life cycle of a lesson](fig/life_cycle.png){alt="Diagram of the life cycle of a
+lesson in The Carpentries ecosystem. A lesson is proposed at the beginning of the 
+pre-alpha stage. It enters alpha when it is taught for the first time. In beta, it is 
+taught by other instructors. A full release of the lesson is made when it is stable. 
+Pilot workshops take place during the alpha and beta phases. The Carpentries Lab hosts 
+open peer review of lessons that have completed the beta phase."}
+
+Each life cycle stage indicates the level of maturity of a lesson:
+
+- **pre-alpha**: a first draft of the lesson is still being constructed.
+- **alpha**: the lesson has been/is being taught by the original authors, but has not been well tested.
+**beta**: the lesson is ready to be taught by instructors who have not been significantly involved in its developed to this point.
+**stable**: the lesson has been extensively tested by the authors and others. It can be considered broadly complete and unlikely to undergo any drastic changes without warning.
```

Comment:
```suggestion
- **pre-alpha**: a first draft of the lesson is still being constructed.
- **alpha**: the lesson has been/is being taught by the original authors, but has not been well tested.
- **beta**: the lesson is ready to be taught by instructors who have not been significantly involved in its developed to this point.
- **stable**: the lesson has been extensively tested by the authors and others. It can be considered broadly complete and unlikely to undergo any drastic changes without warning.
```
missing bullet points